fuse/eval/metrics/metrics_common.py

```diff
@@ -160,7 +160,10 @@ def collect(self, batch: Dict) -> None:
             batch_to_collect = {}
 
             for name, key in self._keys_to_collect.items():
-                value = batch[key]
+                try:
```

Comment:
Was it just for debugging?
Or would you like to print a more informative message and then reraise the exception?